app/players/elves/druid.py

```diff
@@ -0,0 +1,15 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str, musical_instrument: str,
+                 favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        return (f"Druid {self.nickname}. {self.nickname} "
+                + f"has a favourite spell: {self._favourite_spell}")
```

Comment:
The `player_info` method constructs a string with the druid's nickname and favourite spell. Ensure that the `nickname` attribute is accessible from the `Elf` class.